### DISCUSSION TITLE
v5.5.0

### DISCUSSION THREAD

**Discussion by juanpicado** [verdaccio/verdaccio#2951] [Announcements]
## Features

**TL;DR**

* Upgrade [@verdaccio-ui-theme](https://github.com/verdaccio-ui-theme)  ([d36b8b1](https://github.com/verdaccio/verdaccio/commit/d36b8b1e00ed4180424af160dcf0d53697d8a905))
* Use backward compatible UI from v6 on v5 ([#2912](https://github.com/verdaccio/verdaccio/issues/2912)) ([b79266d](https://github.com/verdaccio/verdaccio/commit/b79266d54e7310ed879e80e0beda95ebbeb2470c))
* Update dependencies, include 🔒 security flagged ones.

### Web User Interface Improvements

The current user interface for this version is exactly the same used on the next major release, the [verdaccio/ui](https://github.com/verdaccio/ui) **repository has been archived**, one single UI is much easier to maintain and also to share features between the future major release and the stable one.

The translations were migrated from the source code to the [crowdin platform](https://translate.verdaccio.org/) for the User interface which ensure repeated words are not translated twice and also crowding provides a nice interface for contributions, the latest translations are pulled on every release automcatically. 

* Add new options to the web disable login and pkg managers ([#2947](https://github.com/verdaccio/verdaccio/issues/2947)) ([1a7b1ee](https://github.com/verdaccio/verdaccio/commit/1a7b1ee8ad79c0a164f8bae7160b55919c18c0f1))

  - Users are able to select which package managers you want to enabled on the web user interface ref #2226 
  - Users are able to enable/disable login on web and endpoints option ref #2228 (this is another security layer which some users might be interested to disable)
```yaml
web:
  pkgManagers:
    - npm
    - yarn
    - pnpm
  login: true      
```

#### Improve language switch ui and package manager info 

- Remove the language switch as autocomplete, add info how to contribute to translations
- Convert info dialog to be able host more information, learn how to use registry with Yarn 2 and add links to documentation.

![2022-01-22_13-17](https://user-images.githubusercontent.com/558752/150638261-fa911c9d-0401-4f9c-8fca-b6280df1ac2a.png)
![2022-01-20_22-09](https://user-images.githubusercontent.com/558752/150422510-e30d229c-e6e5-4642-b075-f5fe018dd8ca.png)

## Bug Fixes

> ⚠️ The `marked` dependency has been updated to v4 which was marked a security upgrade through the `@verdaccio/readme` dependency.

* **deps:** update all non-major linting dependencies (5.x) ([#2885](https://github.com/verdaccio/verdaccio/issues/2885)) ([23d0bd7](https://github.com/verdaccio/verdaccio/commit/23d0bd7056333bd87bf232c93f25f6f6e853816f))
* **deps:** update dependency @verdaccio/local-storage to v10.1.1 ([#2932](https://github.com/verdaccio/verdaccio/issues/2932)) ([8c7e39e](https://github.com/verdaccio/verdaccio/commit/8c7e39e8f3547fc59cfabcf6a58276894f8c4e9d))
* **deps:** update dependency @verdaccio/readme to v10.2.1 ([#2933](https://github.com/verdaccio/verdaccio/issues/2933)) ([8a75f8c](https://github.com/verdaccio/verdaccio/commit/8a75f8cc360813a04e3b6c553ca5db4e213c7dd8))
* **deps:** update dependency async to v3.2.3 ([#2934](https://github.com/verdaccio/verdaccio/issues/2934)) ([e2eb6ce](https://github.com/verdaccio/verdaccio/commit/e2eb6ce27feb4097fe9940db46ad67ce99b35547))
* **deps:** update dependency fast-safe-stringify to v2.1.1 ([#2891](https://github.com/verdaccio/verdaccio/issues/2891)) ([bd738ec](https://github.com/verdaccio/verdaccio/commit/bd738ecf6026d70c249aa8a00a32f8e95fe80e8d))
* **deps:** update dependency verdaccio-audit to v10.1.1 ([#2935](https://github.com/verdaccio/verdaccio/issues/2935)) ([3a9d773](https://github.com/verdaccio/verdaccio/commit/3a9d77341596904e0b5f637b4e81fe4037f65d8a))
* update babel and core dependencies ([#2887](https://github.com/verdaccio/verdaccio/issues/2887)) ([ec98159](https://github.com/verdaccio/verdaccio/commit/ec98159c3c5e3e890e749a3d8fd555b4c5c0be63))
* update docker to v14.18.2 ([#2888](https://github.com/verdaccio/verdaccio/issues/2888)) ([22cfea0](https://github.com/verdaccio/verdaccio/commit/22cfea09070a060437fe01a74e9bd12b2cf834d9))

## Announcements

The next month is the talk at Node.js Congress 2022, you can craft your own badge 😊 [here](https://nodecongress.com/badge/juan_picado), please share it with your colleagues and 👍🏼 if you want like this tweet.

<a href="https://twitter.com/NodeCongress/status/1483792936827510785?s=20">
<img width="438" alt="Screenshot 2022-01-22 165430" src="https://user-images.githubusercontent.com/558752/150648064-2193a788-4cfb-4ae3-bac6-f6c6be8ab3dd.png">
</a>

<hr /><em>This discussion was created from the release <a href='https://github.com/verdaccio/verdaccio/releases/tag/v5.5.0'>v5.5.0</a>.</em>
